src/signUp.js

```diff
@@ -0,0 +1,47 @@
+import { TEST_AUTH, ERROR_MESSAGES } from "./constants/VALIDATION.js";
+import {
+  emailError,
+  emailInput,
+  pwError,
+  pwInput,
+  pwConfirmError,
+  pwConfirmInput,
+  form,
+  pwToggle,
+  pwConfirmToggle,
+  validateEmail,
+  validatePw,
+  validatePwConfirm,
+  eyeToggle,
+  showError,
+} from "./auth.js";
+
+const isDuplicateEmail = () => {
+  if (emailInput.value.trim() === TEST_AUTH.email) {
+    showError(emailError, emailInput, ERROR_MESSAGES.email_duplicate);
+    return true;
+  } else return false;
+};
```

Comment:
`validateEmailDuplication`과 같이 동사형으로 지어주셔야 합니다.

Reply:
현재 이메일 중복 검증에 따른 에러 표출 + 중복 여부 반환 두 가지 역할을 하는데, 중복 여부는 애초에 조건으로 사용되기 때문에 따로 전역변수로 만들어서 이 함수와 이 함수의 반환값을 필요로 하는 곳에서 사용하시면 됩니다.
```suggestion
const isEmailDuplicate = emailInput.value.trim() === TEST_AUTH.email;

const validateEmailDuplication = () => {
  if (isEmailDuplicate) {
    showError(emailError, emailInput, ERROR_MESSAGES.email_duplicate);
  }
};
```